pom.xml

```diff
@@ -2,20 +2,42 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.cryptomator</groupId>
 	<artifactId>cli</artifactId>
-	<version>0.6.0-SNAPSHOT</version>
+	<version>1.0.0-SNAPSHOT</version>
```

Comment:
don't 1.0 before we are sure the CLI commands won't change

Reply:
Resolved in https://github.com/cryptomator/cli/pull/68/commits/da44681db80d9a24e0ca0cb40c194aba7dece855